babel/test/data/boards.js

```diff
@@ -651,7 +651,7 @@ export const boardData = {
                 [0, 0, 3]
             ]
         },
-        'swapData': {
+        'swap': {
```

Comment:
Javascript only needs stringified keys when there's spaces, dashes, etc. in the key name.